### ISSUE TITLE
The referenced component 'CefSharp.Wpf' could not be found.

### ISSUE THREAD

**Issue by stephenuwu** [syngp/SynapseX#9]
The referenced component 'CefSharp.Wpf' could not be found.

I am getting this error whenever I try to compile the Synapse UI WPF file.

**Comment by Mellonyt** [comment]
try setting mode to X86 and release

**Comment by stephenuwu** [comment]
Yeah I did that, I'm using VS2019 is that not compatible? I also have GCC Compiler installed but I don't think that is of relevance.

**Comment by Mellonyt** [comment]
Open the SynapseX-Master directory and go to packages and open the attached zip file and drag  and drop the stuff in the zip into the directory
[package.zip](https://github.com/syngp/SynapseX/files/4378818/package.zip)

**Comment by stephenuwu** [comment]
I have to restart my computer to build and I will test this later. I also want to note that I already had the CefSharp stuff in my packages folder and was told to replace the files.
